frontend/src/components/Participants.vue

```diff
@@ -0,0 +1,303 @@
+      if (
+        Object.values(this.getParticipants).find(
+          user => user.phone === $event.target.value
+        )
+      )
+        this.warningAlert = true;
+      this.warningMessage =
+        'This phone number already exists. Please verify if a person is actually a new paricipant';
+      setTimeout(() => (this.warningAlert = false), 10000);
+    }
+  },
+  computed: {
+    ...mapGetters('participant', {
+      getError: 'getError',
+      getParticipants: 'getParticipants'
+    }),
+    emailErrors() {
+      const errors = [];
+
+      if (!this.$v.form.email.$dirty) return errors;
+      !this.$v.form.email.required && errors.push('This field is required');
+      !this.$v.form.email.email && errors.push('Type a correct e-mail address');
+      !this.$v.form.email.emailExists && errors.push('User already exists');
+      return errors;
+    },
+    githubErrors() {
+      const errors = [];
+
+      if (!this.$v.form.github.$dirty) return errors;
+      !this.$v.form.github.alphaNum &&
+        errors.push('You used characters which are not permitted');
+      !this.$v.form.github.required && errors.push('This field is required');
+      !this.$v.form.github.githubExists && errors.push('User already exists');
+      return errors;
+    },
+    firstNameErrors() {
+      const errors = [];
+
+      if (!this.$v.form.first_name.$dirty) return errors;
+      !this.$v.form.first_name.alpha &&
+        errors.push('You used characters which are not permitted');
+      !this.$v.form.first_name.required &&
+        errors.push('This field is required');
+      !this.$v.form.first_name.minLength && errors.push('Name is too short');
+      !this.$v.form.first_name.maxLength && errors.push('Name is too long');
+      return errors;
+    },
+    lastNameErrors() {
+      const errors = [];
+
+      if (!this.$v.form.last_name.$dirty) return errors;
+      !this.$v.form.last_name.alpha &&
+        errors.push('You used characters which are not permitted');
+      !this.$v.form.last_name.required && errors.push('This field is required');
+      !this.$v.form.last_name.minLength &&
+        errors.push('Last name is too short');
+      !this.$v.form.last_name.maxLength && errors.push('Last name is too long');
+      return errors;
+    },
+    slackErrors() {
+      const errors = [];
+
+      if (!this.$v.form.slack.$dirty) return errors;
+      !this.$v.form.slack.alphaNum &&
+        errors.push('You used characters which are not permitted');
+      !this.$v.form.slack.minLength && errors.push('Nick is too short');
+      !this.$v.form.slack.maxLength && errors.push('Nick is too long');
+      return errors;
+    },
+    phoneErrors() {
+      const errors = [];
+
+      if (!this.$v.form.phone.$dirty) return errors;
+      !this.$v.form.phone.numeric &&
+        errors.push('You can only type digits here');
+      !this.$v.form.phone.minLength &&
+        errors.push('Type correct number ie. 111-222-333');
```

Comment:
Minor thing - correct format is 111222333 actual, so that hint is a bit misleading.

Reply:
I corrected the messages. 

I thought about introducing mask, but:
1) it seems it's been[ deprecated in vuetify above 2.0](https://stackoverflow.com/questions/47857673/how-to-use-mask-in-vuetify-text-field) and would require external package like [this](https://vuejs-tips.github.io/vue-the-mask/);
2) Such a mask, changes the value of the field and I'm not sure if we want to store numbers like this 111-222-333 (with dashes). Of course we could replace it before sending, but at this point it seems too much of a hustle.

If you feel like we could do it anyway please create a separate task for introducing vue-the-mask or some other package and replace it both in this form and contact form on the landing page (mask stopped working there).